src/main/java/com/launchdarkly/client/LoggerFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.launchdarkly.client;
+
+import java.util.concurrent.atomic.AtomicReference;
+import org.slf4j.Logger;
+
+public class LoggerFactory {
+
+    private static AtomicReference<ILoggerFactory> factory = new AtomicReference<>();
+
+    public static void install(ILoggerFactory factory) {
```

Comment:
@keynan What do you think about making the LoggerFactory part of the config? Then this method doesn't need to be public, and it makes configuring the Logger consistent with the rest of the LDClient setup.
If there's a reason you need this to be configurable after the client has been created, then I'm curious..
(Depending on your logging framework, log levels can be changed at runtime outside of any code changes to this library)